collector/pkg/collector/metrics.go

```diff
@@ -92,7 +93,7 @@ func (mc *MetricsCollector) Run() error {
 			mc.Collect(device.WWN, device.DeviceName, device.DeviceType)
 
 			if mc.config.GetInt("commands.metrics_smartctl_wait") > 0 {
-				time.Sleep(time.Duration(mc.config.GetInt("commands.metrics_smartctl_wait")) * time.Millisecond)
+				time.Sleep(time.Duration(mc.config.GetInt("commands.metrics_smartctl_wait")) * 1000 * time.Millisecond)
```

Comment:
```suggestion
				time.Sleep(time.Duration(mc.config.GetInt("commands.metrics_smartctl_wait")) * time.Second)
```

Why not just do `time.Second` instead?

Reply:
Well...you are 100% right.
Just changed it!